README.md

```diff
@@ -35,17 +85,62 @@ to the key/value pair. This allows for the garbage collector to remove pairs.
 
 Look at [Map](docs/Map.md) for more information on how to use WeakMap.
 
+### Iterators
+
+The `@dojo/shim/iterator` module is an implementation of the [ES2015 Iterator specification](http://www.ecma-international.org/ecma-262/6.0/#sec-iteration).
+
+### Math
+
+The [`@dojo/shim/math`](docs/math.md) module provides implementations for many math methods.
+
+### Number
 
-## How do I use this package?
+The `dojo/shim/number` module provides implementations for several `Number` methods.
 
-TODO: Add appropriate usage and instruction guidelines
+* `isNaN`
+* `isFinite`
+* `isInteger`
+* `isSafeInteger`
+
+### Object
+
+The `dojo/shim/object` provides implementations of `Object` methods.
+
+* is
+* getOwnPropertySumbols
+* getOwnPropertyNames
+* getOwnPropertyDescriptor
+* values
+* entires
+
+### Observables
+
+The [`@dojo/shim/Observable`](docs/Observable.md) class is an implementation of the proposed [Observable specification](https://tc39.github.io/proposal-observable/).  Observables are further extended in [`@dojo/core/Observable`](https://github.com/dojo/core/blob/master/src/Observable.ts).
+
+### Promises
+
+[`@dojo/shim/Promise`](docs/Promise.md) is an implementation of the [ES2015 Promise specification](http://www.ecma-international.org/ecma-262/6.0/#sec-promise-objects).
+
+### String
+
+The [`@dojo/shim/string`](docs/string.md) module contains `String` methods.
+
+### Symbols
+
+`@dojo/shim/Symbol` provides an implementation of the [ES2015 Symbol specification](http://www.ecma-international.org/ecma-262/6.0/#sec-symbol-objects) for browsers that do not natively support `Symbol`.
```

Comment:
just browsers, or JS environments? (Not that I know of a non-browser JS engine we support that doesn't support Symbols), just don't think the shim is browser specific.